math-scala/src/main/scala/org/apache/mahout/math/drm/logical/CheckpointAction.scala

```diff
@@ -44,5 +44,6 @@ abstract class CheckpointAction[K: ClassTag] extends DrmLike[K] {
     case Some(cp) => cp
   }
 
+  val classTag = implicitly[ClassTag[K]]
```

Comment:
I remember trying to attach it to DrmLike, but I had some problems because it's a trait